iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractVoltageLevel.java

```diff
@@ -24,6 +25,10 @@
  */
 abstract class AbstractVoltageLevel extends AbstractIdentifiable<VoltageLevel> implements VoltageLevelExt {
 
+    private static final int DEFAULT_NODE_INDEX_LIMIT = 1000;
+
+    public static final int NODE_INDEX_LIMIT = loadNodeIndexLimit(PlatformConfig.defaultConfig());
```

Comment:
Isn't there some risks doing that? It means that the `NODE_INDEX_LIMIT` is fixed when creating the voltage level but what if the user changes it? However, not sure if this is really relevant as it is not a parameter that should change a lot... What do you think @flo-dup @geofjamg ?

Reply:
What I had in mind was a high number which would fit in most if not all cases - not sure 1000 is a good enough candidate for that. And then it should be hardly ever changed, only for very specific network with huge voltage levels. Or with voltage levels of any size but poorly numbered, that is, with high indices.

Reply:
Even in this case, shouldn't then this parameter not be static?